themes/index.js

```diff
@@ -71,6 +71,12 @@ const themes = {
     text_color: "f8f8f2",
     bg_color: "282a36",
   },
+  oddextension: {
+    title_color: "EE0BF5",
```

Comment:
Let's prefer lower case values :D also i just noticed in the top of the file there are two Upper case values in bg_color (default, default_repocard) can you please make then lowercase too?

Reply:
Here is the view of my recent theme: used your profile for example

  <img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true&hide_border=false&text_color=641E16&icon_color=145A32&bg_color=EAECEE&title_color=EE0BF5" alt="Profile Stats"/><br>

Reply:
Not going to lie, tbh i did not really liked the color scheme :disappointed_relieved:

Reply:
Okay, np. Let me try another combination then but before that can you tell me which part you don't like most. So, I will work on that to bring a nice scheme

Reply:
The colors does not really match up, that pink with grayish background kinda hurting the eyes & green icons with brown text is also not matching up, maybe try material colors